### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by terrahaxton**
Updated README file to include contact info, license info, and documenation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,26 +5,41 @@ ORD Water Distribution Network Model (AKA "EPANET")
...
-Git Hub and software repositories are geared towards people with software development experience. If you've ever hacked EPANET or built a Matlab script for running hydraulic simulations, you may find the contents of this repository helpful. We encourage you to fork EPANET and develop a great new feature to share with the community. 
+EPANET has an active open source software community including consulting engineers, students, researchers, software companies, professional organizations, other interested members of the public, and EPA partners. An Open Source EPANET Initiative at http://community.wateranalytics.org has released source code for an EPANET engine version 2.1 that included community sourced bug fixes and they are working on additional releases with new feature requests. Others are encouraged to fork EPANET and develop a great new feature to share with the community. 
+
+License
+-------------
+EPANET is released under the Creative Commons CC0 public domain license. See the LICENSE.txt file.
+
+Contributing to EPANET
+-------------
+Information on how to contribute to the development of EPANET can be found in the CONTRIBUTING.md file.
```
Comment (samhatchett) [inline README.md]: is this file part of a PR? i cannot find it

Comment (eladsal) [reply README.md]: I think this PR wasn't for OWA

Comment (samhatchett) [reply README.md]: right, i understand - the readme in this PR refers to `CONTRIBUTING.md`, but i cannot find that file anywhere here. I'm commenting on the USEPA repository as an interested observer 😄

Comment (terrahaxton) [reply README.md]: I was going to put in a new file into the repo called CONTRIBUTING.md, but I haven't done so yet. I plan to push another pull request later today with this file.

**Pull request merged**